test/declaration_test.dart

```diff
@@ -411,13 +411,13 @@ void testMediaQueries() {
 
```

Comment:
Good catch on the repeating expressions must have an AND.  However, there are no tests for the simplest case with one expression e.g.,

@media only screen (min-device-width: 4000px) {
...
}

Do you think its a good idea to have a failure case that is caught when there is more than one expression?